interview-questions.md

```diff
@@ -14,44 +14,59 @@ Researched answer:
 
-Researched answer:
+Researched answer: Indexes are sequential locations in an array, which means the first value in the string is set at 0 and moves to the right, so the next value is set at 1, even though it is the second value in the string. The content in each element in an array is defined as the value. For example, let myNumbers = [1, 2, 3, 4] : the values are "1,2,3,4" while the index of 1 is 0, 2 is 1, 3 is 2, and 4 is 3.  
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: The visual difference between arrays and strings is arrays are defined by square brackets [] and strings are defined by quotations " " , ' ', ` `. The functional difference between arrays and strings is arrays are meant to hold large amounts of data in one defined variable and strings are intended to be a language that communicates code to another human, since the computer does not understand words. Both arrays and strings are indexed at 0. Additionally, they both can hold lots of data types.
 
-Researched answer:
+Researched answer: Strings can be added to arrays ex. let myNames = ["tabi" , "tabitha" "tabitha trubey"]. An array holds a conglomoration of varaibles, as long as they are JS datatypes that can be called on as one variable altogether. A string holds a set of characters defined by quotations.
+Despite many differences, they also share many similarities, like being indexed at 0 and having length properties. You can also convert arrays into strings and strings into arrays by using .join("") and .split("") accessors. 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-Your answer:
+Your answer: Pair programming allows two developers to work on one portion of code to mitigate flaws/bugs and to produce a comprhensive project as two brains are better than one. The proper techniques for pair programming consist of a driver and navigator. The driver is the only one on the keyboard typing code. The navigator informs the driver what to write and how to write the proper syntax. The navigator also corrects any spelling or syntax errors in each line before proceeding to the next line. This ensures an efficient work flow and encourages communication. 
 
-Researched answer:
+Researched answer: Pair programming allows two programmers to work on the same code at the same time from a driver and a navigator position. The navigator tells the driver what code to write and the driver executes the code. This increases communication and prevents hidden agendas or confusion in code, as two people are physically sitting right next to each other talking through the code together. Roles are often switched to allow for equal distribution of work and responsibilities. Pair programming creates efficiency, as expectations are outlined and executed with accountability from each partner. 
```

Comment:
Excellent work in this section as well!  You answers are thorough and explained very well 👏